doc/source/limitation.rst

```diff
@@ -33,44 +24,40 @@ Query log cannot be directed to table.
    Use ``general_log`` and ``general_log_file`` to choose query logging
    and the log file name.
 
-* Maximum allowed transaction size is defined by the
-  :variable:`wsrep_max_ws_rows` and :variable:`wsrep_max_ws_size` variables.
-  ``LOAD DATA INFILE`` processing will commit every 10 000 rows.
-  So large transactions due to ``LOAD DATA``
-  will be split to series of small transactions.
+Maximum allowed transaction size is defined by the |max_ws_rows| and |max_ws_size| variables.
+   ``LOAD DATA INFILE`` processing will commit every 10 000 rows.  So large
+   transactions due to ``LOAD DATA`` will be split to series of small
+   transactions.
 
-* Due to cluster-level optimistic concurrency control,
-  transaction issuing ``COMMIT`` may still be aborted at that stage.
-  There can be two transactions writing to the same rows
-  and committing in separate |PXC| nodes,
-  and only one of the them can successfully commit.
-  The failing one will be aborted.
-  For cluster-level aborts, |PXC| gives back deadlock error code: ::
+Transaction issuing ``COMMIT`` may still be aborted at that stage.
+   Due to cluster-level optimistic concurrency control. There can be two
```

Comment:
"Due to ... control." is not a complete sentence.  I think this should use a comma, not a period.